henchman/helpers.go

```diff
@@ -21,6 +24,16 @@ func MergeMap(src map[interface{}]interface{}, dst map[interface{}]interface{},
 	}
 }
 
+func MergeLogrusFields(src log.Fields, dst log.Fields, override bool) {
```

Comment:
We can just convert VarsMap to a map[string]interface{} type.  However, there are issues with the yaml parser if we do change it away from map[interface{}]interface{}